whylog_vim/gui.py

```diff
@@ -0,0 +1,209 @@
+from whylog import LineSource, FrontInput
+
+from mock import MagicMock, patch
+import os.path
+import vim
+from whylog_vim.input_reader.teacher_reader import get_button_name, filter_comments
+from whylog_vim.consts import WindowTypes, WindowSizes
+
+
+
+class WindowContext(object):
+
+    def __init__(self, gui):
+        self.gui = gui
+
+    def __enter__(self):
+        vim.command('setlocal modifiable')
```

Comment:
Maybe wrapp vim commands somehow?
Maybe craete VimCommander class which has functions that wrap vim.command(something)